--- _server.py	2017-06-22 18:11:53.000000000 +0900
+++ _server2.py	2017-06-22 18:18:27.000000000 +0900
@@ -3,6 +3,7 @@ from requests.packages.urllib3.util.url 
 from slackclient._channel import Channel
 from slackclient._user import User
 from slackclient._util import SearchList, SearchDict
+import ssl
 from ssl import SSLError
 
 from websocket import create_connection
@@ -100,10 +101,12 @@ class Server(object):
             proxy_auth, proxy_port, proxy_host = None, None, None
 
         try:
+            sslopt = {"cert_reqs": ssl.CERT_NONE}
             self.websocket = create_connection(ws_url,
                                                http_proxy_host=proxy_host,
                                                http_proxy_port=proxy_port,
-                                               http_proxy_auth=proxy_auth)
+                                               http_proxy_auth=proxy_auth,
+                                               sslopt=sslopt)
             self.websocket.sock.setblocking(0)
         except Exception as e:
             raise SlackConnectionError(str(e))

